Monofoxe/Monofoxe.Engine/Utils/GameMath.cs

```diff
@@ -208,24 +208,124 @@ public static float GetArea(List<Vector2> vertices)
 		}
 
 
+
+		#region Bezier curve stuff.
+		// A look up table for factorials. Capped to 16.
+		private static float[] Factorial = new float[]
+		{
+			1.0f,
+			1.0f,
+			2.0f,
+			6.0f,
+			24.0f,
+			120.0f,
+			720.0f,
+			5040.0f,
+			40320.0f,
+			362880.0f,
+			3628800.0f,
+			39916800.0f,
+			479001600.0f,
+			6227020800.0f,
+			87178291200.0f,
+			1307674368000.0f,
+			20922789888000.0f,
+		};
+
+		private static float Binomial(int n, int i)

```

Comment:
This method needs at least some description. Google what it does and just copypaste a quick summary or a url to wikipedia.